src/components/EventMapMarker.css

```diff
@@ -0,0 +1,10 @@
+.Event-dot {
```

Comment:
Prefix the classname with `EventMapMarker` so `.EventMapMarker-dot` in this case